examples/express-sample/app.js

```diff
@@ -1,52 +1,60 @@
-var config = require('config');
+var config = require("config");
+
+if (config.Raygun.Key === "YOUR_API_KEY") {
```

Comment:
Is this the only verification we should be doing here? We could also check expected format with pattern-matching for example?

Reply:
Ok, I see this is for an example app, so worries

Reply:
Curious because you brought it up - is there a format for RG API keys? Because if there is, then it'd be useful to add that to other providers as well.

Reply:
This verification is added to the example projects to warn the user that they did not change the default one.

Marking this conversation as resolved as doesnt need an action.